DesktopModules/Vanjaro/UXManager/Extensions/Migrate/Components/HttpModule.cs

```diff
@@ -59,12 +59,14 @@ private void context_PreRequestHandlerExecute(object sender, EventArgs e)
                                 string ControlPanel = HostController.Instance.GetSettings().ContainsKey("ControlPanel") ? HostController.Instance.GetSettings()["ControlPanel"].Value.ToLower() : null;
 
                                 HostController.Instance.GetSettings()["ControlPanel"].Value = "DesktopModules/Vanjaro/UXManager/Library/Base.ascx";
-                                HostController.Instance.GetSettings()["DisableEditBar"].Value = "True";
+                                if (HostController.Instance.GetBoolean("DisableEditBar"))
```

Comment:
@mksmanish79 This doesn't make sense. Can you please review with @narensingh008 

If the value is already true, why set it to true again?

Reply:
naren will update this

Reply:
@mksmanish79 I still don't get this code. Why set it to true if it's already true?